packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/examples/SelectedPeopleList.Basic.DragDrop.Example.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import { IPersona } from 'office-ui-fabric-react/lib/Persona';
+import { people } from '@uifabric/example-data';
+import { SelectedPeopleList } from '@uifabric/experiments/lib/SelectedItemsList';
+import { useSelectedItems } from '../../../UnifiedPicker/hooks/useSelectedItems';
+import { DragDropHelper } from 'office-ui-fabric-react/lib/utilities/dragdrop/DragDropHelper';
+import { getTheme, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { IDragDropEvents, Selection } from 'office-ui-fabric-react';
+
+export const SelectedPeopleListBasicDragDropExample = <T extends {}>(): JSX.Element => {
+  const [index, setIndex] = React.useState(0);
+  const [addMultipleKey, setAddMultipleKey] = React.useState(50);
+  const [currentSelectedItems, setCurrentSelectedItems] = React.useState<IPersona[]>([people[40]]);
+
+  const selection = new Selection();
+  const [draggedItem, setDraggedItem] = React.useState<IPersona>();
+  const [draggedIndex, setDraggedIndex] = React.useState(-1);
+  const dragDropHelper = new DragDropHelper({
+    selection: selection,
+    minimumPixelsForDrag: 5,
+  });
+
+  const { selectedItems, unselectAll, getSelectedItems } = useSelectedItems(selection, currentSelectedItems);
+
+  const theme = getTheme();
+  const dragEnterClass = mergeStyles({
+    backgroundColor: theme.palette.neutralLight,
+  });
+
+  const _insertBeforeItem = (item: T): void => {
+    const draggedItems = selection.isIndexSelected(draggedIndex) ? (getSelectedItems() as T[]) : [draggedItem!];
+    const insertIndex = selectedItems.indexOf(item);
+    const items = selectedItems.filter(itm => draggedItems.indexOf(itm) === -1);
```

Comment:
nit: item :)

Reply:
given that the input name is item, I thought having this be slightly different would be less confusing

Reply:
got it.. i thought the spelling got missed .. :) maybe some other var name would also work

Reply:
I changed it to be less confusing